core/modules/tests.rs

```diff
@@ -691,32 +686,6 @@ async fn dyn_import_ok() {
   .await;
 }
 
-#[tokio::test]
```

Comment:
Was never exported and should not be used regardless.

Reply:
Can you elaborate how it was not safe?

Reply:
It just cleared the module map without clearing the pending modules or any other important stuff. The comments stated that it wasn't safe, and it isn't actually used anywhere so I'm just removing a potential footgun.

The API was only ever tested, so I'm not sure what it was intended for, but we definitely shouldn't be doing anything like this.

Reply:
https://github.com/denoland/deno_core/pull/245

This was a very recent community contribution.

Reply:
I completely forgot about this. IIRC they had a use case for testing and were aware of the footgun-like nature of this API. I think we should keep it after all.